### ISSUE TITLE
[SCM-518] wrong model in vss-settings.mdo works only with a bug in Modello 1.0-alpha-21 but not with Modello 1.0+

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-scm#731]
**[Hervé Boutemy](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=hboutemy)** opened **[SCM-518](https://issues.apache.org/jira/browse/SCM-518?redirect=false)** and commented

after [documentation](http://maven.apache.org/scm/vss.html), vss-settings.xml file content is

```xml
<vss-settings>
  <Settings>
    <vssDirectory>ss_exe_directory</vssDirectory>
  </Settings>
</vss-settings>
```

but actual vss-settings.mdo does not define "Settings" field.

Reading such a file works only because of a bug in Modello parsing: MODELLO-152, which has been fixed in Modello 1.0.

There are 2 options for us: either change the documentation (and tell the users) to support following model (which was the initial intend IMHO):

```xml
<vss-settings>
  <vssDirectory>ss_exe_directory</vssDirectory>
</vss-settings>
```

or tweak the model and update the code to create an intermediate class for this Settings element (only useful to avoid changing the XML format)

---

**Affects:** 1.2

1 votes, 0 watchers

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

This issue has been auto closed because it has been inactive for a long period of time. If you think this issue still applies, retest your problem with the most recent version of Maven and the affected component, reopen and post your results.

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

Fixed with [368968b814ec983f2e504be0907ad2187148b594](https://gitbox.apache.org/repos/asf?p=maven-scm.git;a=commit;h=368968b814ec983f2e504be0907ad2187148b594).
